59-bottlenose-dolphin/technical_specification.md

```diff
@@ -0,0 +1,121 @@
+# Angular implementation plan and migration strategy (Bottlenose Dolphin)
+
+**Epic Type:** Exploratory Epic
+
+## Principle Components of Exploration:
+
+As outlined in [ADR002](https://github.com/ghga-de/adrs/blob/main/docs/adrs/adr002_angular_as_frontend_framework.md), we decided to migrate our existing GHGA data portal from React to Angular.
+
+This exploratory epic is concerned with laying out a plan for achieving this goal that includes the most important architecture and design decisions that should be made upfront. We expect that these may be revised later, but we want to start with a solid and reasonable foundation to achieve a maintainable solution with our limited resources.
+
+The following questions need to be investigated and answered in this context:
+
+- Architecture and modularization:
+  - Can or should we vertically slice the application?
+  - Do we want to integrate future frontend applications like for GHGA SPE and Atlas as sub-applications under a single host application?
```

Comment:
I think this needs also needs some elaboration. What would a sub-application structure entail? Is a 'single host application' equal to an angular project?

Reply:
Rephrased this a bit in c149bb1f and put it after the question about micro frontends.